src/main/java/codeu/model/data/Hashtag.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codeu.model.data;
+
+import java.time.Instant;
+import java.util.UUID;
+
+/** Class representing a hashtag. */
+public class Hashtag {
+
+  private final UUID id;
+  private final UUID ownerId;
+  private final Instant creation;
+  private final String content;
+  private final Boolean createdFromUser;
```

Comment:
Based on the comment on `getOwnerId`, it looks like this boolean is used to differentiate between hashtags created by the user vs. a conversation, and thus indicates who/what the "owner" refers to.  Since these are really separate choices, and it seems possible that there could be more choices in the future, I'd make this an `enum` called something like HashtagCreator with values like USER and CONVERSATION (or just CONVO or something shorter).